lib/airrecord/table.rb

```diff
@@ -43,7 +43,7 @@ def find(id)
         end
       end
 
-      def records(filter: nil, sort: nil, view: nil, offset: nil, paginate: true, fields: nil)
+      def records(filter: nil, sort: nil, view: nil, offset: nil, paginate: true, fields: nil, maxrecords: nil, pagesize: nil)
```

Comment:
Let's put an underscore between the words here, i.e. `max_records` and `page_size`.